tests/build/build.sh

```diff
@@ -0,0 +1,148 @@
+#!/bin/bash
+
+WORK_DIR=/opt/openssl/tests/build
+REPO_DIR=${WORK_DIR}/openssl
+TOOLSREPO_DIR=""
+FLAG_FILE=${WORK_DIR}/RUNNING
+[ -f ${FLAG_FILE} ] && echo "The build task is already running. Quitting." && exit 1 
+# default metrics values
+ALLOWED_OSSL_VERSIONS=("master" "1.1.1" "3.0" "3.1")
+OSSL_BASE_GIT_LINK="https://github.com/openssl"
+OSSL_GIT_LINK="${OSSL_BASE_GIT_LINK}/openssl"
+OSSLTOOLS_GIT_LINK="${OSSL_BASE_GIT_LINK}/tools"
+TOOLS=0
+declare -A OSSL_GIT_VERSIONS
+OSSL_GIT_VERSIONS[master]="master"
+OSSL_GIT_VERSIONS[1.1.1]="OpenSSL_1_1_1-stable"
+OSSL_GIT_VERSIONS[3.0]="openssl-3.0"
+OSSL_GIT_VERSIONS[3.1]="openssl-3.1"
```

Comment:
Should we include _important_ releases too?
E.g. the FIPS validated releases (3.0.0 and 3.0.8)?

Just a thought.

Reply:
They are static, what would periodic testing of these be good for?